Doesn't work with browserify?

```
Cannot find module './ReactDefaultPerf' from '/Users/contra/Projects/project-name/node_modules/draft-js-hashtag-plugin/lib'
```

It looks like the output being published to npm is specifically meant to only work with webpack, because browserify hits these require statements and blows up.

--- a/draft-js-emoji-plugin/.babelrc
+++ b/draft-js-emoji-plugin/.babelrc
@@ -1,0 +1,16 @@
+{
+  "presets": ["react", "es2015", "stage-0"],
+  "env": {
+    "production": {
+      "plugins": [
+        [
+          "babel-plugin-webpack-loaders",
+          {
+            "config": "${WEBPACK_CONFIG}",
+            "verbose": true
+          }
+        ]
+      ]
+    }
+  }
+}

--- a/draft-js-emoji-plugin/package.json
+++ b/draft-js-emoji-plugin/package.json
@@ -27,7 +27,10 @@
     "react-dom": ">=0.14.0 || 15.0.0-rc.2"
   },
   "scripts": {
-    "build": "../node_modules/.bin/rimraf lib && NODE_ENV=production BABEL_ENV=production ../node_modules/.bin/webpack --config webpack.config.js",
+    "clean": "../node_modules/.bin/rimraf lib",
+    "build": "npm run clean && npm run build:js && npm run build:css",
+    "build:js": "WEBPACK_CONFIG=$(pwd)/webpack.config.js BABEL_DISABLE_CACHE=1 BABEL_ENV=production NODE_ENV=production ../node_modules/.bin/babel --out-dir='lib' --ignore='__test__/*' src",
+    "build:css": "node ../scripts/concatCssFiles $(pwd) && ../node_modules/.bin/rimraf lib-css",
     "prepublish": "npm run build"
   },
   "license": "MIT",

--- a/draft-js-emoji-plugin/webpack.config.js
+++ b/draft-js-emoji-plugin/webpack.config.js
@@ -1,57 +1,24 @@
 /* eslint-disable no-var */
 var path = require('path');
-var webpack = require('webpack');
 var ExtractTextPlugin = require('extract-text-webpack-plugin');
 var autoprefixer = require('autoprefixer');
 
 module.exports = {
-  devtool: 'source-map',
-
-  entry: {
-    index: ['./src/index.js'],
-  },
-
   output: {
-    path: path.join(__dirname, 'lib'),
-    filename: '[name].js',
-    libraryTarget: 'commonjs2',
     publicPath: '/',
-  },
-
-  resolve: {
-    extensions: ['', '.js'],
+    libraryTarget: 'commonjs2', // necessary for the babel plugin
+    path: path.join(__dirname, 'lib-css'), // where to place webpack files
   },
   module: {
     loaders: [
       {
-        test: /\.js$/,
-        loaders: ['babel'],
-        include: [
-          path.join(__dirname, 'src'),
-        ],
-      }, {
         test: /\.css$/,
-        loader: ExtractTextPlugin.extract('style-loader', 'css-loader?modules&importLoaders=1&localIdentName=[local]___[hash:base64:5]!postcss-loader'),
-        include: [
-          path.join(__dirname, 'src'),
-        ],
+        loader: ExtractTextPlugin.extract('style-loader', 'css-loader?modules&importLoaders=1&localIdentName=draftJsEmojiPlugin__[local]__[hash:base64:5]!postcss-loader'),
       },
     ],
   },
   postcss: [autoprefixer({ browsers: ['> 1%'] })],
-
   plugins: [
-    new ExtractTextPlugin('plugin.css', { allChunks: true }),
-    new webpack.optimize.OccurenceOrderPlugin(),
-    new webpack.DefinePlugin({
-      'process.env': {
-        NODE_ENV: JSON.stringify('production'),
-      },
-    }),
-
-    // new webpack.optimize.UglifyJsPlugin({
-    //   screw_ie8: true,
-    //   compressor: { warnings: false },
-    // }),
+    new ExtractTextPlugin(`${path.parse(process.argv[2]).name}.css`),
   ],
 };

--- a/draft-js-hashtag-plugin/.babelrc
+++ b/draft-js-hashtag-plugin/.babelrc
@@ -1,0 +1,16 @@
+{
+  "presets": ["react", "es2015", "stage-0"],
+  "env": {
+    "production": {
+      "plugins": [
+        [
+          "babel-plugin-webpack-loaders",
+          {
+            "config": "${WEBPACK_CONFIG}",
+            "verbose": true
+          }
+        ]
+      ]
+    }
+  }
+}

--- a/draft-js-hashtag-plugin/package.json
+++ b/draft-js-hashtag-plugin/package.json
@@ -27,7 +27,9 @@
     "react-dom": ">=0.14.0 || 15.0.0-rc.2"
   },
   "scripts": {
-    "build": "../node_modules/.bin/rimraf lib && NODE_ENV=production BABEL_ENV=production ../node_modules/.bin/webpack --config webpack.config.js",
+    "clean": "../node_modules/.bin/rimraf lib",
+    "build": "npm run clean && npm run build:js",
+    "build:js": "WEBPACK_CONFIG=$(pwd)/webpack.config.js BABEL_DISABLE_CACHE=1 BABEL_ENV=production NODE_ENV=production ../node_modules/.bin/babel --out-dir='lib' --ignore='__test__/*' src",
     "prepublish": "npm run build"
   },
   "license": "MIT",

--- a/draft-js-hashtag-plugin/package.json
+++ b/draft-js-hashtag-plugin/package.json
@@ -28,8 +28,9 @@
   },
   "scripts": {
     "clean": "../node_modules/.bin/rimraf lib",
-    "build": "npm run clean && npm run build:js",
+    "build": "npm run clean && npm run build:js && npm run build:css",
     "build:js": "WEBPACK_CONFIG=$(pwd)/webpack.config.js BABEL_DISABLE_CACHE=1 BABEL_ENV=production NODE_ENV=production ../node_modules/.bin/babel --out-dir='lib' --ignore='__test__/*' src",
+    "build:css": "node ../scripts/concatCssFiles $(pwd) && ../node_modules/.bin/rimraf lib-css",
     "prepublish": "npm run build"
   },
   "license": "MIT",

--- a/draft-js-hashtag-plugin/webpack.config.js
+++ b/draft-js-hashtag-plugin/webpack.config.js
@@ -7,7 +7,7 @@ module.exports = {
   output: {
     publicPath: '/',
     libraryTarget: 'commonjs2', // necessary for the babel plugin
-    path: path.join(__dirname, 'lib'), // where to place webpack files
+    path: path.join(__dirname, 'lib-css'), // where to place webpack files
   },
   module: {
     loaders: [
@@ -19,6 +19,6 @@ module.exports = {
   },
   postcss: [autoprefixer({ browsers: ['> 1%'] })],
   plugins: [
-    new ExtractTextPlugin('plugin.css', { allChunks: true }),
+    new ExtractTextPlugin(`${path.parse(process.argv[2]).name}.css`),
   ],
 };

--- a/draft-js-hashtag-plugin/webpack.config.js
+++ b/draft-js-hashtag-plugin/webpack.config.js
@@ -1,57 +1,24 @@
 /* eslint-disable no-var */
 var path = require('path');
-var webpack = require('webpack');
 var ExtractTextPlugin = require('extract-text-webpack-plugin');
 var autoprefixer = require('autoprefixer');
 
 module.exports = {
-  devtool: 'source-map',
-
-  entry: {
-    index: ['./src/index.js'],
-  },
-
   output: {
-    path: path.join(__dirname, 'lib'),
-    filename: '[name].js',
-    libraryTarget: 'commonjs2',
     publicPath: '/',
-  },
-
-  resolve: {
-    extensions: ['', '.js'],
+    libraryTarget: 'commonjs2', // necessary for the babel plugin
+    path: path.join(__dirname, 'lib'), // where to place webpack files
   },
   module: {
     loaders: [
       {
-        test: /\.js$/,
-        loaders: ['babel'],
-        include: [
-          path.join(__dirname, 'src'),
-        ],
-      }, {
         test: /\.css$/,
-        loader: ExtractTextPlugin.extract('style-loader', 'css-loader?modules&importLoaders=1&localIdentName=[local]___[hash:base64:5]!postcss-loader'),
-        include: [
-          path.join(__dirname, 'src'),
-        ],
+        loader: ExtractTextPlugin.extract('style-loader', 'css-loader?modules&importLoaders=1&localIdentName=draftJsHashtagPlugin__[local]__[hash:base64:5]!postcss-loader'),
       },
     ],
   },
   postcss: [autoprefixer({ browsers: ['> 1%'] })],
-
   plugins: [
     new ExtractTextPlugin('plugin.css', { allChunks: true }),
-    new webpack.optimize.OccurenceOrderPlugin(),
-    new webpack.DefinePlugin({
-      'process.env': {
-        NODE_ENV: JSON.stringify('production'),
-      },
-    }),
-
-    // new webpack.optimize.UglifyJsPlugin({
-    //   screw_ie8: true,
-    //   compressor: { warnings: false },
-    // }),
   ],
 };

--- a/draft-js-linkify-plugin/.babelrc
+++ b/draft-js-linkify-plugin/.babelrc
@@ -1,0 +1,16 @@
+{
+  "presets": ["react", "es2015", "stage-0"],
+  "env": {
+    "production": {
+      "plugins": [
+        [
+          "babel-plugin-webpack-loaders",
+          {
+            "config": "${WEBPACK_CONFIG}",
+            "verbose": true
+          }
+        ]
+      ]
+    }
+  }
+}

--- a/draft-js-linkify-plugin/package.json
+++ b/draft-js-linkify-plugin/package.json
@@ -27,7 +27,10 @@
     "react-dom": ">=0.14.0 || 15.0.0-rc.2"
   },
   "scripts": {
-    "build": "../node_modules/.bin/rimraf lib && NODE_ENV=production BABEL_ENV=production ../node_modules/.bin/webpack --config webpack.config.js",
+    "clean": "../node_modules/.bin/rimraf lib",
+    "build": "npm run clean && npm run build:js && npm run build:css",
+    "build:js": "WEBPACK_CONFIG=$(pwd)/webpack.config.js BABEL_DISABLE_CACHE=1 BABEL_ENV=production NODE_ENV=production ../node_modules/.bin/babel --out-dir='lib' --ignore='__test__/*' src",
+    "build:css": "node ../scripts/concatCssFiles $(pwd) && ../node_modules/.bin/rimraf lib-css",
     "prepublish": "npm run build"
   },
   "license": "MIT",

--- a/draft-js-linkify-plugin/webpack.config.js
+++ b/draft-js-linkify-plugin/webpack.config.js
@@ -1,57 +1,24 @@
 /* eslint-disable no-var */
 var path = require('path');
-var webpack = require('webpack');
 var ExtractTextPlugin = require('extract-text-webpack-plugin');
 var autoprefixer = require('autoprefixer');
 
 module.exports = {
-  devtool: 'source-map',
-
-  entry: {
-    index: ['./src/index.js'],
-  },
-
   output: {
-    path: path.join(__dirname, 'lib'),
-    filename: '[name].js',
-    libraryTarget: 'commonjs2',
     publicPath: '/',
-  },
-
-  resolve: {
-    extensions: ['', '.js'],
+    libraryTarget: 'commonjs2', // necessary for the babel plugin
+    path: path.join(__dirname, 'lib-css'), // where to place webpack files
   },
   module: {
     loaders: [
       {
-        test: /\.js$/,
-        loaders: ['babel'],
-        include: [
-          path.join(__dirname, 'src'),
-        ],
-      }, {
         test: /\.css$/,
-        loader: ExtractTextPlugin.extract('style-loader', 'css-loader?modules&importLoaders=1&localIdentName=[local]___[hash:base64:5]!postcss-loader'),
-        include: [
-          path.join(__dirname, 'src'),
-        ],
+        loader: ExtractTextPlugin.extract('style-loader', 'css-loader?modules&importLoaders=1&localIdentName=draftJsLinkifyPlugin__[local]__[hash:base64:5]!postcss-loader'),
       },
     ],
   },
   postcss: [autoprefixer({ browsers: ['> 1%'] })],
-
   plugins: [
-    new ExtractTextPlugin('plugin.css', { allChunks: true }),
-    new webpack.optimize.OccurenceOrderPlugin(),
-    new webpack.DefinePlugin({
-      'process.env': {
-        NODE_ENV: JSON.stringify('production'),
-      },
-    }),
-
-    // new webpack.optimize.UglifyJsPlugin({
-    //   screw_ie8: true,
-    //   compressor: { warnings: false },
-    // }),
+    new ExtractTextPlugin(`${path.parse(process.argv[2]).name}.css`),
   ],
 };

--- a/draft-js-mention-plugin/.babelrc
+++ b/draft-js-mention-plugin/.babelrc
@@ -1,0 +1,16 @@
+{
+  "presets": ["react", "es2015", "stage-0"],
+  "env": {
+    "production": {
+      "plugins": [
+        [
+          "babel-plugin-webpack-loaders",
+          {
+            "config": "${WEBPACK_CONFIG}",
+            "verbose": true
+          }
+        ]
+      ]
+    }
+  }
+}

--- a/draft-js-mention-plugin/package.json
+++ b/draft-js-mention-plugin/package.json
@@ -27,7 +27,10 @@
     "react-dom": ">=0.14.0 || 15.0.0-rc.2"
   },
   "scripts": {
-    "build": "../node_modules/.bin/rimraf lib && NODE_ENV=production BABEL_ENV=production ../node_modules/.bin/webpack --config webpack.config.js",
+    "clean": "../node_modules/.bin/rimraf lib",
+    "build": "npm run clean && npm run build:js && npm run build:css",
+    "build:js": "WEBPACK_CONFIG=$(pwd)/webpack.config.js BABEL_DISABLE_CACHE=1 BABEL_ENV=production NODE_ENV=production ../node_modules/.bin/babel --out-dir='lib' --ignore='__test__/*' src",
+    "build:css": "node ../scripts/concatCssFiles $(pwd) && ../node_modules/.bin/rimraf lib-css",
     "prepublish": "npm run build"
   },
   "license": "MIT",

--- a/draft-js-mention-plugin/webpack.config.js
+++ b/draft-js-mention-plugin/webpack.config.js
@@ -1,57 +1,24 @@
 /* eslint-disable no-var */
 var path = require('path');
-var webpack = require('webpack');
 var ExtractTextPlugin = require('extract-text-webpack-plugin');
 var autoprefixer = require('autoprefixer');
 
 module.exports = {
-  devtool: 'source-map',
-
-  entry: {
-    index: ['./src/index.js'],
-  },
-
   output: {
-    path: path.join(__dirname, 'lib'),
-    filename: '[name].js',
-    libraryTarget: 'commonjs2',
     publicPath: '/',
-  },
-
-  resolve: {
-    extensions: ['', '.js'],
+    libraryTarget: 'commonjs2', // necessary for the babel plugin
+    path: path.join(__dirname, 'lib-css'), // where to place webpack files
   },
   module: {
     loaders: [
       {
-        test: /\.js$/,
-        loaders: ['babel'],
-        include: [
-          path.join(__dirname, 'src'),
-        ],
-      }, {
         test: /\.css$/,
-        loader: ExtractTextPlugin.extract('style-loader', 'css-loader?modules&importLoaders=1&localIdentName=[local]___[hash:base64:5]!postcss-loader'),
-        include: [
-          path.join(__dirname, 'src'),
-        ],
+        loader: ExtractTextPlugin.extract('style-loader', 'css-loader?modules&importLoaders=1&localIdentName=draftJsMentionPlugin__[local]__[hash:base64:5]!postcss-loader'),
       },
     ],
   },
   postcss: [autoprefixer({ browsers: ['> 1%'] })],
-
   plugins: [
-    new ExtractTextPlugin('plugin.css', { allChunks: true }),
-    new webpack.optimize.OccurenceOrderPlugin(),
-    new webpack.DefinePlugin({
-      'process.env': {
-        NODE_ENV: JSON.stringify('production'),
-      },
-    }),
-
-    // new webpack.optimize.UglifyJsPlugin({
-    //   screw_ie8: true,
-    //   compressor: { warnings: false },
-    // }),
+    new ExtractTextPlugin(`${path.parse(process.argv[2]).name}.css`),
   ],
 };

--- a/draft-js-plugins-editor/.babelrc
+++ b/draft-js-plugins-editor/.babelrc
@@ -1,0 +1,3 @@
+{
+  "presets": ["react", "es2015", "stage-0"]
+}

--- a/draft-js-sticker-plugin/.babelrc
+++ b/draft-js-sticker-plugin/.babelrc
@@ -1,0 +1,16 @@
+{
+  "presets": ["react", "es2015", "stage-0"],
+  "env": {
+    "production": {
+      "plugins": [
+        [
+          "babel-plugin-webpack-loaders",
+          {
+            "config": "${WEBPACK_CONFIG}",
+            "verbose": true
+          }
+        ]
+      ]
+    }
+  }
+}

--- a/draft-js-sticker-plugin/package.json
+++ b/draft-js-sticker-plugin/package.json
@@ -27,7 +27,10 @@
     "react-dom": ">=0.14.0 || 15.0.0-rc.2"
   },
   "scripts": {
-    "build": "../node_modules/.bin/rimraf lib && NODE_ENV=production BABEL_ENV=production ../node_modules/.bin/webpack --config webpack.config.js",
+    "clean": "../node_modules/.bin/rimraf lib",
+    "build": "npm run clean && npm run build:js && npm run build:css",
+    "build:js": "WEBPACK_CONFIG=$(pwd)/webpack.config.js BABEL_DISABLE_CACHE=1 BABEL_ENV=production NODE_ENV=production ../node_modules/.bin/babel --out-dir='lib' --ignore='__test__/*' src",
+    "build:css": "node ../scripts/concatCssFiles $(pwd) && ../node_modules/.bin/rimraf lib-css",
     "prepublish": "npm run build"
   },
   "license": "MIT",

--- a/draft-js-sticker-plugin/webpack.config.js
+++ b/draft-js-sticker-plugin/webpack.config.js
@@ -1,57 +1,24 @@
 /* eslint-disable no-var */
 var path = require('path');
-var webpack = require('webpack');
 var ExtractTextPlugin = require('extract-text-webpack-plugin');
 var autoprefixer = require('autoprefixer');
 
 module.exports = {
-  devtool: 'source-map',
-
-  entry: {
-    index: ['./src/index.js'],
-  },
-
   output: {
-    path: path.join(__dirname, 'lib'),
-    filename: '[name].js',
-    libraryTarget: 'commonjs2',
     publicPath: '/',
-  },
-
-  resolve: {
-    extensions: ['', '.js'],
+    libraryTarget: 'commonjs2', // necessary for the babel plugin
+    path: path.join(__dirname, 'lib-css'), // where to place webpack files
   },
   module: {
     loaders: [
       {
-        test: /\.js$/,
-        loaders: ['babel'],
-        include: [
-          path.join(__dirname, 'src'),
-        ],
-      }, {
         test: /\.css$/,
-        loader: ExtractTextPlugin.extract('style-loader', 'css-loader?modules&importLoaders=1&localIdentName=[local]___[hash:base64:5]!postcss-loader'),
-        include: [
-          path.join(__dirname, 'src'),
-        ],
+        loader: ExtractTextPlugin.extract('style-loader', 'css-loader?modules&importLoaders=1&localIdentName=draftJsStickerPlugin__[local]__[hash:base64:5]!postcss-loader'),
       },
     ],
   },
   postcss: [autoprefixer({ browsers: ['> 1%'] })],
-
   plugins: [
-    new ExtractTextPlugin('plugin.css', { allChunks: true }),
-    new webpack.optimize.OccurenceOrderPlugin(),
-    new webpack.DefinePlugin({
-      'process.env': {
-        NODE_ENV: JSON.stringify('production'),
-      },
-    }),
-
-    // new webpack.optimize.UglifyJsPlugin({
-    //   screw_ie8: true,
-    //   compressor: { warnings: false },
-    // }),
+    new ExtractTextPlugin(`${path.parse(process.argv[2]).name}.css`),
   ],
 };

--- a/draft-js-undo-plugin/.babelrc
+++ b/draft-js-undo-plugin/.babelrc
@@ -1,0 +1,16 @@
+{
+  "presets": ["react", "es2015", "stage-0"],
+  "env": {
+    "production": {
+      "plugins": [
+        [
+          "babel-plugin-webpack-loaders",
+          {
+            "config": "${WEBPACK_CONFIG}",
+            "verbose": true
+          }
+        ]
+      ]
+    }
+  }
+}

--- a/draft-js-undo-plugin/package.json
+++ b/draft-js-undo-plugin/package.json
@@ -27,7 +27,10 @@
     "react-dom": ">=0.14.0 || 15.0.0-rc.2"
   },
   "scripts": {
-    "build": "../node_modules/.bin/rimraf lib && NODE_ENV=production BABEL_ENV=production ../node_modules/.bin/webpack --config webpack.config.js",
+    "clean": "../node_modules/.bin/rimraf lib",
+    "build": "npm run clean && npm run build:js && npm run build:css",
+    "build:js": "WEBPACK_CONFIG=$(pwd)/webpack.config.js BABEL_DISABLE_CACHE=1 BABEL_ENV=production NODE_ENV=production ../node_modules/.bin/babel --out-dir='lib' --ignore='__test__/*' src",
+    "build:css": "node ../scripts/concatCssFiles $(pwd) && ../node_modules/.bin/rimraf lib-css",
     "prepublish": "npm run build"
   },
   "license": "MIT",

--- a/draft-js-undo-plugin/webpack.config.js
+++ b/draft-js-undo-plugin/webpack.config.js
@@ -1,57 +1,24 @@
 /* eslint-disable no-var */
 var path = require('path');
-var webpack = require('webpack');
 var ExtractTextPlugin = require('extract-text-webpack-plugin');
 var autoprefixer = require('autoprefixer');
 
 module.exports = {
-  devtool: 'source-map',
-
-  entry: {
-    index: ['./src/index.js'],
-  },
-
   output: {
-    path: path.join(__dirname, 'lib'),
-    filename: '[name].js',
-    libraryTarget: 'commonjs2',
     publicPath: '/',
-  },
-
-  resolve: {
-    extensions: ['', '.js'],
+    libraryTarget: 'commonjs2', // necessary for the babel plugin
+    path: path.join(__dirname, 'lib-css'), // where to place webpack files
   },
   module: {
     loaders: [
       {
-        test: /\.js$/,
-        loaders: ['babel'],
-        include: [
-          path.join(__dirname, 'src'),
-        ],
-      }, {
         test: /\.css$/,
-        loader: ExtractTextPlugin.extract('style-loader', 'css-loader?modules&importLoaders=1&localIdentName=[local]___[hash:base64:5]!postcss-loader'),
-        include: [
-          path.join(__dirname, 'src'),
-        ],
+        loader: ExtractTextPlugin.extract('style-loader', 'css-loader?modules&importLoaders=1&localIdentName=draftJsUndoPlugin__[local]__[hash:base64:5]!postcss-loader'),
       },
     ],
   },
   postcss: [autoprefixer({ browsers: ['> 1%'] })],
-
   plugins: [
-    new ExtractTextPlugin('plugin.css', { allChunks: true }),
-    new webpack.optimize.OccurenceOrderPlugin(),
-    new webpack.DefinePlugin({
-      'process.env': {
-        NODE_ENV: JSON.stringify('production'),
-      },
-    }),
-
-    // new webpack.optimize.UglifyJsPlugin({
-    //   screw_ie8: true,
-    //   compressor: { warnings: false },
-    // }),
+    new ExtractTextPlugin(`${path.parse(process.argv[2]).name}.css`),
   ],
 };

--- a/package.json
+++ b/package.json
@@ -48,6 +48,7 @@
     "babel-jest": "^9.0.3",
     "babel-loader": "^6.2.4",
     "babel-plugin-react-transform": "^2.0.2",
+    "babel-plugin-webpack-loaders": "^0.3.10",
     "babel-preset-es2015": "^6.6.0",
     "babel-preset-react": "^6.5.0",
     "babel-preset-react-hmre": "^1.1.1",

--- a/scripts/concatCssFiles.js
+++ b/scripts/concatCssFiles.js
@@ -1,0 +1,13 @@
+const fs = require('fs');
+const path = require('path');
+const dirname = path.join(process.argv[2], 'lib-css'); // where to place webpack files
+var content = ''; // eslint-disable-line no-var
+
+const filenames = fs.readdirSync(dirname);
+filenames.forEach((filename) => {
+  const filePath = path.join(dirname, filename);
+  content += fs.readFileSync(filePath, 'utf-8');
+});
+
+const outputFilePath = path.join(process.argv[2], 'lib', 'plugin.css');
+fs.writeFileSync(outputFilePath, content);

